pom.xml

```diff
@@ -74,7 +74,7 @@
     <spring-boot.version>2.7.11</spring-boot.version>
 
     <!-- Application dependencies -->
-    <kuflow.version>2.3.1</kuflow.version>
+    <kuflow.version>2.3.2</kuflow.version>
```

Comment:
You must wait to release the SDK before merging this PR and update this

Reply:
Yes

Reply:
But I installed in local and tested

Reply:
but this will be the final version?